### DISCUSSION TITLE
Any guides to setting up websearch?

### DISCUSSION THREAD

**Discussion by keithsol91** [open-webui/open-webui#6242] [Q&A]
Im not very technical but I got a dedicated server setup and then installed Cloudron and then Openwebui -- Got my openai and openrouter API connected

I am not trying to enable websearch. Specifically I need it to search the web and get results back for things like "Best tv in 2024" 

I would also like to give it a specific blog post and then ask it to come up with some social media content for that blog post. Is that doable?

**Comment by White1706rus** [comment]
Found any solution?

**Reply by richtong** [reply]
It's pretty complicated and the documentation is all over the place, but https://docs.openwebui.com/tutorials/integrations/web_search/ explains how to enable a web search engine (and yes there are a sea of these. You can also follow https://tongfamily.com/b/H0n where I'm trying to take notes about how to use all this stuff.
